test/unit/region.spec.js

```diff
@@ -122,18 +122,17 @@ describe('region', function() {
         onSwapOut: function() {}
       });
 
-      this.MyView = Backbone.View.extend({
+      this.MyView = Marionette.View.extend({
```

Comment:
I thought it was just a Backbone view for simplicity, not to actually test that case. It seemed to fit in with these existing tests, and I didn't know conventions around making new test blocks.